README.md

```diff
@@ -72,3 +73,14 @@
 - [Basic Java Interview Questions](https://javaconceptoftheday.com/top-25-simple-basic-java-interview-questions-for-freshers/) - Top 25 Simple Basic Java Interview Questions For Freshers.
 - [Reverse Interview](https://github.com/viraptor/reverse-interview/blob/master/README.md) - List of interview questions (general and technical) you ask the interviewer.
 - [Tech Interview Handbook](https://github.com/yangshun/tech-interview-handbook) - Curated content to help you ace your next technical/behavioral interview with a focus on algorithms, system design, and front-end. 
+
+## Additional Resources
+- [GitHub Student Developer Pack](https://education.github.com/pack) - Details about all the offers
+- What offers you can get with Student Developer Pack: 
+-[FrontendMasters](https://frontendmasters.com/welcome/github-student-developers/) - 6 months of subscription 
+-[JetBrains](https://www.jetbrains.com/community/education/#students) - JetBrains and many more
+
+### Requirements
```

Comment:
Let's remove this section and leave it for folks to read on the Github webpage instead.